src/containers/Tenant/Diagnostics/Configs/Configs.tsx

```diff
@@ -0,0 +1,159 @@
+            return Number(!Current) + Name.toLowerCase();
+        },
+    },
+    {
+        name: 'Default',
+        get header() {
+            return i18n('td-default');
+        },
+        render: ({row}) => {
+            switch (row.Default) {
+                case true:
+                    return i18n('enabled');
+                case false:
+                    return i18n('disabled');
+                default:
+                    return '-';
+            }
+        },
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+    {
+        name: 'Current',
+        get header() {
+            return i18n('td-current');
+        },
+        render: ({row}) => <Switch disabled checked={(row.Current ?? row.Default) || false} />,
+        width: 100,
+        sortable: false,
+        resizeable: false,
+    },
+];
+
+interface FeatureFlagsListsProps {
+    isError?: boolean;
+    featureFlags: FeatureFlagConfig[];
+    featureFlagsFilter?: string;
+}
+
+const FeatureFlagsList = ({isError, featureFlags, featureFlagsFilter}: FeatureFlagsListsProps) => {
+    if (isError) {
+        return i18n('error-msg');
+    }
+
+    if (!featureFlags.length) {
+        return i18n(featureFlagsFilter ? 'search-empty' : 'no-data');
+    }
+
+    return (
+        <ResizeableDataTable
+            columnsWidthLSKey={FEATURE_FLAGS_COLUMNS_WIDTH_LS_KEY}
+            columns={columns}
+            data={featureFlags}
+            settings={DEFAULT_TABLE_SETTINGS}
+        />
+    );
+};
+
+interface ConfigsProps {
+    database: string;
+}
+
+export const Configs = ({database}: ConfigsProps) => {
+    const dispatch = useTypedDispatch();
+    const [search, setSearch] = useQueryParam('search', StringParam);
+    const {currentData = [], isFetching, isError} = tenantApi.useGetClusterConfigQuery({database});
+    const featureFlagsFilter = useTypedSelector(
+        (state) => state.tenant.featureFlagsFilter,
+    )?.toLocaleLowerCase();
+
+    // sync store and query filter
+    React.useEffect(() => {
+        if (search) {
+            dispatch(setFeatureFlagsFilter(search));
+        } else if (featureFlagsFilter) {
+            setSearch(featureFlagsFilter, 'replaceIn');
+        }
+    }, []);
+
+    const onChange = (value: string) => {
+        dispatch(setFeatureFlagsFilter(value));
+        setSearch(value || undefined, 'replaceIn');
+    };
+
+    const featureFlags = featureFlagsFilter
+        ? currentData.filter((item) => item.Name.toLocaleLowerCase().includes(featureFlagsFilter))
+        : currentData;
+
+    return (
+        <TableWithControlsLayout>
+            <TableWithControlsLayout.Controls>
+                <Search
+                    value={featureFlagsFilter}
+                    onChange={onChange}
+                    placeholder={i18n('search-placeholder')}
+                />
+            </TableWithControlsLayout.Controls>
+            <TableWithControlsLayout.Table loading={isFetching}>
```

Comment:
Let's move this and fetching and filtering data to the `FeatureFlagsList`.

Reply:
Why? Now `Configs` component includes logic and `FeatureFlagsList ` is stupid (and is not very necessary imho). After moving logic `Config` will be almost redundant

Reply:
discussed in pm, fixed